WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -528,8 +528,8 @@ private extension NotificationDetailsViewController {
 //
 private extension NotificationDetailsViewController {
     func attachSuggestionsViewIfNeeded() {
-        guard shouldAttachSuggestionsView else {
-            suggestionsTableView.removeFromSuperview()
+        guard shouldAttachSuggestionsView, let suggestionsTableView = self.suggestionsTableView else {
+            self.suggestionsTableView?.removeFromSuperview()
```

Comment:
You `guard let` this just above, so better use the local guarded unwrapped binding directly here

```suggestion
            suggestionsTableView.removeFromSuperview()
```

Reply:
The code that comes inside the **else braces** does not reference the guarded unwrapped, actually.

We're gonna need to either move the guard above (suggested below), ooor restore the `?`, otherwise the crash still happens.

Thank you both gentlemen!!